docs/5g-on-helium/mobile-poc.mdx

```diff
@@ -371,13 +445,158 @@ Latency     150 ->  `Fail`
 Speed Test Tier ->  `Fail`        -> 0.0x Multiplier
 ```
 
-### Statistics on Helium Explorer
+</TabItem>
+</Tabs>
+
+### Speed Test Example Calculations
+
+Here are a set of tables to help visualize the components that make up the Speed Test result.
+
+:::info
+
+It may take up to 12 hours for newly connected 5G Hotspots to perform a Speed Test, and a minimum of
+2 Speed Tests are required for the MOBILE PoC Oracle to calculate a moving average.
+
+:::
+
+<Tabs block={true}>
+
+<TabItem value="successful speed test" label="Successful Speed Test" default>
```

Comment:
Would it make sense to add status icons like you have for other toggle buttons for different outcomes?

Reply:
yes, good call, adding icons and reordering the tabs to move from "good" -> "not good" as well